app/controllers/portfolios_controller.rb

```diff
@@ -1,2 +1,8 @@
 class PortfoliosController < ApplicationController
+    before_action :authenticate_user!
+    def index
+        portfolios = current_user.portfolios
+        # byebug
```

Comment:
Don't leave your byebug in here!

Reply:
Thanks for catching that, we will take it out.